test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+ansible==2.5.3.0
```

Comment:
maybe it's better to define an ansible >= 2.5.5.0

Reply:
Updated